docs/scenario_editor/scenario_files.md

```diff
@@ -4,16 +4,20 @@ By opening the Scenario Tool Widget, users can save scenarios to monoDrive scena
 
 The AI vehicle behavior details and tags that can be altered to create the scenario file can be found in [Scenario Creation](../scenarios)
 
-## Scenario Tool Widget
+## Scenario File Tool Widget
 
 To begin, find the under the Content Folder.
```

Comment:
"find the tool under the Content Folder, right-click and select Run Editor Utility Widget "